File: PRAKTIKUM_5/PRAK504-2410817210004-Rahmad.py
def reverse(A):
    rev_Num=0
    while A!=0:
        remainder=A%10
        rev_Num=rev_Num*10+remainder
        A=A//10
    return rev_Num
A=int(input())
B=int(input())
A=reverse(A)
B=reverse(B)
C=A+B
print(reverse(C))

File: PRAKTIKUM_5/PRAK504-2410817210004-Rahmad.c
#include <stdio.h>
int reverse(int a){
// Lengkapi Function ini
 int remainder,angka=a,rev_Num=0;
 while (angka != 0){
         remainder = angka % 10;

        rev_Num = rev_Num * 10 + remainder;

         angka = angka/10;
 }
 return rev_Num ;
}
int main() {
int A, B;
scanf("%d %d",&A,&B);
A=reverse(A);
B=reverse(B);
int C = A+B;
printf("%d",reverse(C));
}